src/libsync/bandwidthmanager.cpp

```diff
@@ -147,7 +147,7 @@ void BandwidthManager::unregisterDownloadJob(QObject *o)
 
 void BandwidthManager::relativeUploadMeasuringTimerExpired()
 {
-    if (!usingRelativeUploadLimit() || _relativeUploadDeviceList.count() == 0) {
+    if (!usingRelativeUploadLimit() || _relativeUploadDeviceList.size() == 0) {
```

Comment:
I would like to take this opportunity to favor a change to: `|| _relativeUploadDeviceList.empty()`

Reply:
I concur. Thanks for pointing this out. Done with d6f9bd00174af3eb7ee05ac72a7e3695a351a548.